license.java

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright (c) Connexta
+
+This is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as
+published by the Free Software Foundation, either version 3
+of the License, or any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details. A copy of the
+GNU Lesser General Public License is distributed along with this
+program and can be found at http://www.gnu.org/licenses/lgpl.html.
```

Comment:
Is this the right comment formatting for a java header? I would have expected it to look like
```java
/*
 * ...
 */
```

Reply:
🤦‍♂ nice catch